css/styles.css

```diff
@@ -0,0 +1,36 @@
+body {
+  font-family: 'Heebo', sans-serif;
+}
+
+label {
+  display: block;
+  margin: 10px 0;
+}
+
+header {
+  background-color: black;
+  color: white;
+  padding: 10px 20px;
+}
+
+input,
+textarea,
+img {
+  width: 100%;
+}
+
+.submit-button {
+  margin-left: 85%;
+  margin-bottom: 5%;
+  box-shadow: 1px 1px 1px black;
+}
+
+input {
+  border-color: black;
+}
+
+.image-box-submit > input,
+.title-field > input {
+  height: 35px;
+  width: 245%;
```

Comment:
😱 😱 😱 😱 😱 

What purpose is this serving? You definitely don't want to use values over 100%

Reply:
That was to expand the input field since it was really small at 100%, I changed it to pixels though.